IPython/html/static/base/js/dialog.js

```diff
@@ -20,6 +21,9 @@ define([
         var dialog_content = $("<div/>")
             .addClass("modal-content")
             .appendTo(dialog);
+        if(typeof(options.body) === 'string'){
+            options.body = $("<p/>").text(security.sanitize_html(options.body))
```

Comment:
There's no reason to use `.text` *and* `sanitize_html`, since `.text` escapes everything.